python/tk_flame_review/ui/submit_dialog.py

```diff
@@ -17,46 +17,44 @@
     if isinstance(cls, type): globals()[name] = cls
 
 
-
 from  . import resources_rc
 
 class Ui_SubmitDialog(object):
     def setupUi(self, SubmitDialog):
         if not SubmitDialog.objectName():
-            SubmitDialog.setObjectName("SubmitDialog")
+            SubmitDialog.setObjectName(u"SubmitDialog")
```

Comment:
I don't understand why we have those `u` changes since the repo was already compiled with PySide2 script @eduardoChaucaGallegos

Reply:
Seems like this has always been the case and it doesn't make sense.

I see in the deleted `build_resources.sh` file this line:

```bash
sed -i"" -E 's/u\"/\"/g' $UI_PYTHON_PATH/$3.py
```

Which, If I understood correctly, removes the `u` from strings. Seems like we need to add this to the build script?

Reply:
Ok thanks @carlos-villavicencio-adsk That makes sense now.

Do we? Is it worth it?
I mean if the pyside script choose to generate "u" strings, why should we sed it?

The only reason I see is to reduce the size of our PRs

Reply:
@eduardoChaucaGallegos @carlos-villavicencio-adsk and myself made a decision on that: let's rely on PySide script default u strings instead of a regexp that could fail somewhere.